gradle.properties

```diff
@@ -19,7 +19,7 @@ android.useAndroidX=true
 
 #Project Settings
 # Change debugPrivateData to true for debugging
-vector.debugPrivateData=false
+vector.debugPrivateData=true
 # httpLogLevel values: NONE, BASIC, HEADERS, BODY
-vector.httpLogLevel=BASIC
+vector.httpLogLevel=BODY
```

Comment:
You should have kept the comment here: https://github.com/vector-im/element-android/commit/fe1d9aa589e248507555c4243361dccdb459a8c8#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19L23 to avoid editing that